### PR TITLE
implement modal utils for injected script

### PR DESCRIPTION
**Pull request by scriptfans**
Usage example:

First, visit any website, then open chrome console, run `window.testModal()`, you can see the test modal appears in the page.

image

### CODE REVIEW

**Comment by dwalintukan** [comment]
@scriptfans is this for the transaction confirmation dialog?

File: static/manifest.json [hunk 1]
```diff
@@ -59,7 +59,8 @@
     "commons.contentscript-inpage.js",
     "commons.popup-inpage.js",
     "inpage.js",
-    "images/logo.png"
+    "images/logo.png",
+    "modal.css"
```
Comment (dwalintukan) [inline static/manifest.json]: can you move `modal.css` to  `css/modal.css`?

**Comment by scriptfans** [comment]
@dwalintukan Yes, we can use it to interact with DApp. For example, choose a specific wallet, or send tx confirmation.

File: static/manifest.json (hunk 1 above)
Comment (scriptfans) [reply static/manifest.json]: done.

**Comment by dwalintukan** [comment]
@scriptfans please delete your branches after they get merged in. thanks

**Pull request merged**